src/VisualStudio/CSharp/Impl/Options/CSharpSettingsManagerOptionSerializer.cs

```diff
@@ -120,7 +122,8 @@ protected override string GetStorageKeyForOption(IOption option)
         protected override bool SupportsOption(IOption option, string languageName)
         {
             if (option == OrganizerOptions.PlaceSystemNamespaceFirst ||
-                option == OrganizerOptions.WarnOnBuildErrors ||
+                option == AddImportOptions.SuggestForTypesInReferenceAssemblies ||
+                option == AddImportOptions.SuggestForTypesInNuGetPackages ||
```

Comment:
Should we rename `OrganizerOptions` or put these together in the same type (`ImportOptions`?)